.circle/install-mongo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+sudo apt-get purge mongodb-org*
```

Comment:
Also, keep in mind that the way circle works, containers would block until all the parallel steps are finished so there is no gain in only installing mongo on some containers but not the other.

As far as ramdisk goes, I would need your explanation on why this change is needed and how it should affect the build. I'd rather we do it in a separate PR.